docs/coordinates/formatting.rst

```diff
@@ -37,9 +37,9 @@ more). For example::
 You can also use Python's `format` string method to create more complex
 string expressions, such as IAU-style coordinates or even full sentences::
 
-  >>> 'SDSS J{0}{1}'.format(c.ra.to_string(unit=u.hourangle, sep='', precision=2, pad=True), c.dec.to_string(sep='', precision=2, alwayssign=True, pad=True))
+  >>> f'SDSS J{c.ra.to_string(unit=u.hourangle, sep="", precision=2, pad=True)}{c.dec.to_string(sep="", precision=2, alwayssign=True, pad=True)}'
   'SDSS J123049.42+122328.03'
-  >>> 'The galaxy M87, at an RA of {0.ra.hour:.2f} hours and Dec of {0.dec.deg:.1f} degrees, has an impressive jet.'.format(c)
+  >>> f'The galaxy M87, at an RA of {c.ra.hour:.2f} hours and Dec of {c.dec.deg:.1f} degrees, has an impressive jet.'
```

Comment:
unrelated, but it would be so great to get rid of these single letter variables, `c` could easily be `coord`, or `coo`. If this is all about preaching good practices :)

Reply:
I chose `coo` because `coord` is sometimes used in `from astropy import coordinates as coord`.